questions_three/graphql/graphql_client.py

```diff
@@ -0,0 +1,27 @@
+import json
+
+from .operation_failed import OperationFailed
+from .graphql_response import GraphqlResponse
+
+
+class GraphqlClient:
+
+    def __init__(self, *, http_client, url):
+        self.http_client = http_client
+        self.url = url
+
+    def execute(self, operation, **variables):
+        """
+        :param operation: A string literal containing either a GraphQL query or mutation
+        :param variables: Key=value pairs representing required variables for the GraphQL operation
+        :return: A GraphQlResponse object on a successful request, an OperationFailed object when errors are detected in
+        the response
+        """
+        response = self.http_client.post(
+            url=self.url,
+            headers={'Content-Type': 'application/json'},
```

Comment:
When should we run all this through Black?

Reply:
I guess that would depend on the rest of the repo getting run as well. So, I would imagine, after it gets merged in? :)

Reply:
Alright, the answer for this is now settled. @egabe05 will Black-ify separately, as part of https://cybergrx.atlassian.net/browse/INT-138.